src/core/filters/filterregistry.js

```diff
@@ -25,6 +25,22 @@ export default class FilterRegistry {
     this.availableFieldIds = availableFieldIds;
   }
 
+  /**
+   * Returns an array containing all of the filternodes stored in global storage.
+   * @returns {Array<FilterNode>}
+   */
+  getAppliedFilterNodes () {
```

Comment:
Can we rename this getAllFIlterNodes()? "Applied" filters are at a different level of abstraction that what the registry operates on. It's just a repository for filters.